libs/glfw/build.zig

```diff
@@ -77,17 +87,15 @@ pub fn link(b: *Builder, step: *std.build.LibExeObjStep, options: Options) LinkE
     }
 }
 
-pub const BuildError = error{CannotEnsureDependency} || std.mem.Allocator.Error;
-fn buildLibrary(b: *Builder, mode: std.builtin.Mode, target: std.zig.CrossTarget, options: Options) BuildError!*std.build.LibExeObjStep {
+pub const Buildror = error{CannotEnsureDependency} || std.mem.Allocator.Error;
```

Comment:
`Buildror`? :smile:

Reply:
I honestly have no idea how I managed to do that :sob: :laughing: